src/experiments.js

```diff
@@ -185,25 +172,44 @@ export function toggleExperiment(win, experimentId, opt_on,
 /**
  * Returns a set of experiment IDs currently on.
  * @param {!Window} win
- * @return {!Array<string>}
+ * @return {!Object<string, boolean>}
+ * @visibleForTesting
  */
-function getExperimentIds(win) {
+export function getExperimentTogglesFromCookie(win) {
```

Comment:
Lately, I've been directing to keep the original method private to the module, but instead creating a specifically test method, e.g.
```
function getExperimentTogglesFromCookie(win) {
  ...
 ..
  ..
}

export function getExperimentToglesFromCookieForTesting(win) {
  return getExperimentTogglesFromCookie(win);
}
```

"xForTesting" methods are stripped, so we get a better protection.